core/controllers/base_test.py

```diff
@@ -519,22 +519,39 @@ def test_every_method_has_decorator(self):
                 continue
 
             if handler.get != base.BaseHandler.get:
-                handlers_checked.add(handler)
-                self.assertTrue(hasattr(handler.get, '__wrapped__'))
+                if hasattr(handler.get, '__wrapped__'):
+                    handlers_checked.append((handler.__name__, 'GET', True))
+                else:
+                    handlers_checked.append((handler.__name__, 'GET', False))
 
             if handler.post != base.BaseHandler.post:
-                handlers_checked.add(handler)
-                self.assertTrue(hasattr(handler.post, '__wrapped__'))
+                if hasattr(handler.post, '__wrapped__'):
+                    handlers_checked.append((handler.__name__, 'POST', True))
+                else:
+                    handlers_checked.append((handler.__name__, 'POST', False))
 
             if handler.put != base.BaseHandler.put:
-                handlers_checked.add(handler)
-                self.assertTrue(hasattr(handler.put, '__wrapped__'))
+                if hasattr(handler.put, '__wrapped__'):
+                    handlers_checked.append((handler.__name__, 'PUT', True))
+                else:
+                    handlers_checked.append((handler.__name__, 'PUT', False))
 
             if handler.delete != base.BaseHandler.delete:
-                handlers_checked.add(handler)
-                self.assertTrue(hasattr(handler.delete, '__wrapped__'))
-
+                if hasattr(handler.delete, '__wrapped__'):
+                    handlers_checked.append((handler.__name__, 'DELETE', True))
+                else:
+                    handlers_checked.append(
+                        (handler.__name__, 'DELETE', False))
+
+        self.log_line('Verifying decorators for handlers .... ')
+        for (name, method, status) in handlers_checked:
```

Comment:
The third arg should be `handler_is_decorated` or similar. True/False isn't a status.